gef.py

```diff
@@ -7746,6 +7753,9 @@ def context_source(self):
         except Exception:
             return
 
+        file_base_name = os.path.basename(symtab.filename)
+        bp_locations = [b.location for b in gdb.breakpoints() if file_base_name in b.location]
```

Comment:
I wonder if calling `gdb.breakpoints` is expensive. Maybe we should profile (cc @fibonascii) how much this slows down drawing the context when we have source and a bunch of BPs